src/components/poolstats/PoolPieChartWidget.tsx

```diff
@@ -255,110 +303,121 @@ export default function PoolPieChartWidget(props: PoolStatsWidgetProps) {
       : ` and ${drawData.silo1Label}`
   );
 
-  const mouseIsInsidePieChart = mouseData.isActive && mouseData.r < 1;
+  const firstHalfOfSlices = slices.slice(0, slices.length / 2).reverse();
```

Comment:
Is this split between first half and second half of slices really necessary?

Reply:
With the current implementation/simplification, yes. It is one of the tradeoffs made. The reason is that I want it to be able to support different numbers of slices, which hardcoding would not allow.

Reply:
iirc it already supported different numbers of slices?